kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/gpio/KuraGPIOTrigger.java

```diff
@@ -13,7 +13,7 @@
 package org.eclipse.kura.gpio;
 
 public enum KuraGPIOTrigger {
-    RAISING_EDGE,
+    RISING_EDGE,
```

Comment:
Hello and thanks for the contribution!
This is a braking change to the APIs. 
It would be better if you could deprecate the old option and add the new one.
Implementation-wise you should try to keep the compatibility with the old option since it is deprecated but still there.

Thanks
Best regards,
Matteo

Reply:
Hi Matteo and thank you for the feedback!
Ok then I will add the deprecate annotation for the old option.

Kind regards,
Marco